binary.dat

```diff
@@ -0,0 +1 @@
+
```

Comment:
I assume this file was added to the PR by mistake.

Reply:
Removed.